### ISSUE TITLE
Example of user / password login?

### ISSUE THREAD

**Issue by poblish** [quickfixgo/examples#41]
Is there a Golang example of using user and password? Git history suggests there used to be, but it was removed at some point.

We've been using this happily with v.0.6.0, but in v0.7.0 it has no effect and login never occurs (no error). Our previously working code:
```
if msgType, err := message.Header.GetString(tag.MsgType); err == nil && msgType == "A" {
        message.Body.SetBool(tag.ResetSeqNumFlag, true)
        message.Body.SetField(tag.Username, quickfix.FIXString(e.username))
        message.Body.SetField(tag.Password, quickfix.FIXString(e.password))
}
```

Other examples I can find on the 'net suggest this should still work, but it would nice to be sure.

**Comment by ackleymi** [comment]
> Git history suggests there used to be

Can you point me to the relevant history? I can't seem to find it

**Comment by poblish** [comment]
> > Git history suggests there used to be
> 
> Can you point me to the relevant history? I can't seem to find it

Yes, it's impossible to find via Git history. I think it was probably [this message](https://github.com/quickfixgo/quickfix/issues/152#issuecomment-225859962) I originally had in mind, though reading it again I see that the commenter was suggesting new code to add to the existing template (also missing). There's some more context [here](https://github.com/quickfixgo/quickfix/issues/277).

Either way, both of those are 6+ years old, so it would be great to have some guidance as to whether those are still the correct patterns to be using.
